content/deploy/install/services.md

```diff
@@ -178,75 +180,131 @@ Install Tempo with these steps:
     ```
 
 1. Modify the Helm file as needed.
+
 1. Install with Helm:
 
     ```bash
     helm install tempo -f tempo.yaml grafana/tempo -n {{< param application_name >}}
     ```
 
-### Core
+> Note: Depending on your configuration you may need to run Tempo in distributed mode. When installing with Helm instead of using `grafana/tempo` instead do `grafana/tempo-distributed`.
```

Comment:
```suggestion
> [!NOTE]
> Depending on your configuration you may need to run Tempo in distributed mode. When installing with Helm instead of using `grafana/tempo` instead do `grafana/tempo-distributed`.
```